package.json

```diff
@@ -556,12 +556,12 @@
     ],
     "configuration": {
       "type": "object",
-      "title": "MSSQL configuration",
+      "title": "%mssql.Configuration%",
       "properties": {
         "mssql.azureActiveDirectory": {
           "type": "string",
           "default": "AuthCodeGrant",
-          "description": "Chooses which Authentication method to use",
+          "description": "%mssql.chooseAuthMethod%",
           "enum": [
```

Comment:
While you're here could you add enum descriptions for these?

Reply:
@Charles-Gagnon what do you mean by enum descriptions?

Reply:
https://github.com/microsoft/azuredatastudio/blob/a2ded8bd3c5c53a371e344b9b7c729b8adf540ca/extensions/azcli/package.json#L49

Shows up as extra text describing the option 

![image](https://user-images.githubusercontent.com/28519865/144676434-9500e62d-df15-4ef7-9113-9dc1a67b857b.png)